### PR TITLE
Basic email sent with Mailgun with a .env file

### PR DESCRIPTION
**Pull request by BenNewman90**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+mailgunApiKey = bcf56ec9023e62c8767b08c7ce3b0dab-c8e745ec-5150a51a
```
Comment (matthall8) [inline .env]: This file should not be uploaded to GitHub. May need to updated .gitignore to ignore this. Please can you also send the .env contents on Gitter?

File: package.json [hunk 2]
```diff
@@ -8,13 +8,17 @@
...
     "express": "^4.16.3",
+    "fs": "0.0.1-security",
     "gatsby": "^2.0.0",
     "gatsby-plugin-manifest": "^2.0.2",
     "gatsby-plugin-offline": "^2.0.5",
     "gatsby-plugin-react-helmet": "^3.0.0",
     "gatsby-plugin-styled-components": "^3.0.0",
     "gatsby-source-filesystem": "^2.0.1",
     "gatsby-transformer-remark": "^2.1.6",
+    "googleapis": "^27.0.0",
```
Comment (matthall8) [inline package.json]: Do we need GoogleAPIs package? Can't see where we are using it?

File: package.json [hunk 3]
```diff
@@ -8,13 +8,17 @@
     "body-parser": "^1.18.3",
     "dotenv": "^6.0.0",
     "express": "^4.16.3",
+    "fs": "0.0.1-security",
```
Comment (matthall8) [inline package.json]: FS is a built-in component I believe, I don't think this is what you think it is.

File: package.json [hunk 4]
```diff
@@ -8,13 +8,17 @@
...
     "gatsby": "^2.0.0",
     "gatsby-plugin-manifest": "^2.0.2",
     "gatsby-plugin-offline": "^2.0.5",
     "gatsby-plugin-react-helmet": "^3.0.0",
     "gatsby-plugin-styled-components": "^3.0.0",
     "gatsby-source-filesystem": "^2.0.1",
     "gatsby-transformer-remark": "^2.1.6",
+    "googleapis": "^27.0.0",
+    "mailgun-js": "^0.21.0",
+    "netlify-cms": "^2.1.2",
```
Comment (DenisKent) [inline package.json]: We are using netlify-cms package anymore I don't think. If you merge with master is should go away, along with its use.

Comment (matthall8) [reply package.json]: @BenNewman90 - did you see Denis' comment? Once this is fixed it looks good to me.

**Comment by DenisKent** [comment]
It won't let me merge while package-lock is conflicted :(

**Pull request closed without merging**